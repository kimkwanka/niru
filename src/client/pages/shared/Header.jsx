import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () =>
  (
    <header>
      <div className="container flex">
        <div className="logo-text">niru</div>
        <nav>
          <ul>
            <li><NavLink to="/" exact activeClassName="navItemActive">Home</NavLink></li>
            <li><NavLink to="/about" activeClassName="navItemActive">About</NavLink></li>
            <li><NavLink to="/styleguide" activeClassName="navItemActive">Styleguide</NavLink></li>
          </ul>
        </nav>
      </div>
    </header>
  );

export default Header;
