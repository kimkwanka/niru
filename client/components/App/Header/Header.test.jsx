import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './Header';

describe('<Header /> - Snapshot', () => {
  it('matches its snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
