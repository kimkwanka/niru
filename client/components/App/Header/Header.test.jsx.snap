// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> - Snapshot matches its snapshot 1`] = `
<header>
  <div
    className="container flex"
  >
    <div
      className="logo-text"
    >
      niru
    </div>
    <nav>
      <ul>
        <li>
          <NavLink
            activeClassName="navItemActive"
            aria-current="page"
            exact={true}
            to="/"
          >
            Home
          </NavLink>
        </li>
        <li>
          <NavLink
            activeClassName="navItemActive"
            aria-current="page"
            to="/about"
          >
            About
          </NavLink>
        </li>
        <li>
          <NavLink
            activeClassName="navItemActive"
            aria-current="page"
            to="/styleguide"
          >
            Styleguide
          </NavLink>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
